### ISSUE TITLE
File permissions on transcoded files

### ISSUE THREAD

**Issue by klogg416** [ntodd/video_transcoding_docker#7]
Hi, I am having a challenge with file permissions on transcoded file. unRAID uses a single user: group pair (`nobody:users`, `99:100`) for all files, however this container creates files as `root:root` with permissions `0666`.

I have tried passing environment variables as I do with some of the other containers, but they aren't honored and possibly the container needs to expect them to take advantage?

This is my modified run line which still creates files as `root:root` `0666`:
`docker run -it -v "`pwd`":/data -e UMASK=001 -e PUID=99 -e PGID=100 ntodd/video-transcoding`

Any thoughts on how I can get the files created as the correct `user:group` so the other system software and containers can work with them (other than doing a permissions clean up manually at the end)?

Thank you for pulling this container together and keeping it current with Don's releases.
Kyle

**Comment by ntodd** [comment]
Hi, @klogg416. This isn't something I have run in to before, but does the `-u` flag help with your situation?

https://docs.docker.com/engine/reference/#user

Maybe try something like: `docker run -it -v "pwd":/data -u 99:100 ntodd/video-transcoding`

**Comment by klogg416** [comment]
Hi @ntodd, thanks for the quick reply. I was able to test out the `-u` flag and it solved it once I noticed the back ticks were missing in around "pwd" (which I mention in case someone else finds this and looking for a solution). 

```
docker run -it -v "`pwd`":/data -u 99:100 ntodd/video-transcoding
```

I used `touch` to create a new file, and outside the container the permissions were as follows:
```
-rw-r--r-- 1 nobody  users   0 Jul 17 13:23 makefile.txt
```
"nobody" is user 99
"users" is group 100

Thanks for the quick and easy fix!
